### DISCUSSION TITLE
Github und Kaggle CSV Datei unterscheiden sich: disney_plus_titles.csv

### DISCUSSION THREAD

**Discussion by kaiserog** [irgroup-classrooms/dis08-2021#75] [Q&A] [answered]
Guten Tag,

Ich möchte hiermit folgenden Hinweis geben:

Die Datei disney_plus_titles.csv unterscheidet sich in der Anzahl an Zeilen (1369 vs 1450) von [Github](https://github.com/irgroup-classrooms/dis08-2021/blob/main/datasets/disney_plus_titles.csv) zu [Kaggle](https://www.kaggle.com/shivamb/disney-movies-and-tv-shows?select=disney_plus_titles.csv).

Damit kommt es bei der Aufgabe - **Exercise 1 (8 pts) - Assignment 2**  - zu unterschiedlichen Ergebnissen.

Beste Grüße,
Konstantin Kaiser

**Comment by HaakFab** [comment, marked as answer]
Hallo,
ich vermute stark, dass die Daten auf Kaggle regelmäßig aktualisiert werden. Verwenden Sie daher bitte zur besseren Vergleichbarkeit die Daten aus unserem Github Repository.
Sollte es dennoch zu Auffälligkeiten und unterschiedlichen Lösungen kommen, weiss ich jetzt dank Ihres Hinweises, wo ich zuerst nach möglichen Ursachen zu schauen habe! 

Vielen Dank, liebe Grüße und einen erfolgreichen "Endspurt",
Fabian Haak
